lib/Ed25519VerificationKey2020.js

```diff
@@ -293,3 +332,16 @@ function _isEqualBuffer(buf1, buf2) {
   }
   return true;
 }
+
+// check a multibase key for an expected header
+function _isValidKeyHeader(multibaseKey, expectedHeader) {
+  if((typeof multibaseKey !== 'string') ||
+    (multibaseKey[0] !== MULTIBASE_BASE58BTC_HEADER)) {
+    return false;
+  }
```

Comment:
Note: The extra parens are unnecessary around each comparison, but regardless, the preferred style is to make the combined positive assertion (which does require parens) and then not it:

```suggestion
  if(!(typeof multibaseKey === 'string' &&
    multibaseKey[0] === MULTIBASE_BASE58BTC_HEADER)) {
    return false;
  }
```